Src/BootCamp.Chapter/Checks.cs

```diff
@@ -16,26 +16,26 @@ public static class Checks
     {
         public static int PromptInt(string message)
         {
-            // To do: call your implementation. 
-            return 0;
+            return Lesson3.PromptInt(message);
+      
         }
 
         public static string PromptString(string message)
         {
             // To do: call your implementation. 
-            return "";
+            return Lesson3.PromptString(message);
```

Comment:
this what was asked in the homework , create a class called Lesson3 and after that call it from check.cs can i insert a suggestion when i do the modification that u asked instead of pull request ?

Reply:
Tbh, I didn't read the homework.
In which case, what you have done here is fine 👍